src/locales/de/Export.json

```diff
@@ -4,13 +4,16 @@
         "Text": "Sie können wählen, ob der komplette Datensatz, die angezeigte Statistik oder beides herunterladen werden soll."
     },
     "Checkbox": {
-        "DataSet": "Ausgewählte Einzeldaten (.csv)",
-        "Stat": "Ausgewählter aggregierter Datensatz (.csv)", 
-        "Chart": "Diagramm (.png)"
+        "DataSet": "Ausgewählte Einzeldaten",
+        "Csv": "(.csv)",
+        "Isolates": "Isolate",
+        "Stat": "Ausgewählter aggregierter Datensatz (.csv)",
+        "Chart": "Grafik (.png)"

```

Comment:
Would be more clean to use the same pattern for the different labels:
```
"Raw": "Ausgewählte Einzeldaten",
"Raw_Isolates": "Isolate",
"Stat": "Ausgewählter aggregierter Datensatz",
"Chart": "Grafik",
"Csv": "(.csv)",
"Png": "(.png)"
```

Reply:
Done. Commit  "Little changes & Rename".